JAX-RPC RI is not buildable with FastInfoset version > 1.0.2

Later version of FastInfoset  causes build error:
```
    [javac] /home/jenkins/workspace/jax-rpc-ri-master-build/jaxrpc-ri/src/com/sun/xml/rpc/streaming/FastInfosetReader.java:288: error: cannot find symbol
    [javac]         final int end = i + _charactersLength;
    [javac]                             ^
    [javac]   symbol:   variable _charactersLength
    [javac]   location: class FastInfosetReader
```
Class header says `class FastInfosetReader extends StAXDocumentParser` and in `private boolean isWhiteSpaceCharacters()` reference to parent `_charactersLength` is used, but this variable does not exist in later versions.

--- a/jaxrpc-ri/src/com/sun/xml/rpc/streaming/FastInfosetReader.java
+++ b/jaxrpc-ri/src/com/sun/xml/rpc/streaming/FastInfosetReader.java
@@ -167,7 +167,7 @@ public final class FastInfosetReader extends StAXDocumentParser implements XMLRe
                 case EOF :
                     return state;
                 case CHARS :
-                    if (_characters != null && !isWhiteSpaceCharacters()) {
+                    if (_characters != null && !isWhiteSpace()) {
                         return CHARS;
                     }
                     continue;
@@ -281,17 +281,6 @@ public final class FastInfosetReader extends StAXDocumentParser implements XMLRe
         return new XMLReaderException(
             "xmlreader.ioException",
             new LocalizableExceptionAdapter(e));
-    }
-    
-    private boolean isWhiteSpaceCharacters() {
-        int i = _charactersOffset;
-        final int end = i + _charactersLength;
-        while (i < end) {
-            if (_characters[i++] > '\u0020') {
-                return false;
-            }
-        }
-        return true;
     }
     
     

